### PR TITLE
feat: Docker Container Working

### PR DESCRIPTION
**Pull request by ReBy298**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+services:
+  frontend:
+    build:
+      context: ./Front-End
+      dockerfile: Dockerfile
+    ports:
+      - "3000:3000"
+    container_name: frontend_container
```
Comment (mayralgr) [inline docker-compose.yml:10]: give it a more descriptive name, if this goes to a image repos this name is too generic

**Comment by mayralgr** [comment]
Back-End and Front-End follows any casing? I would recommend pick one of the most used ones
https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/
camel and pascal are the most but you can pick one.
The Back-End and Front-End may cause issues with git as by default doesn't take in count casing

**Comment by mayralgr** [comment]
Please add a description on the changes

File: Back-End/src/main/java/com/flight_search/dto/FlightDTO.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
+package com.flight_search.dto;
```
Comment (mayralgr) [inline Back-End/src/main/java/com/flight_search/dto/FlightDTO.java:1]: flight_search
FlightDTO
there is no consistency in the files/directories naming

File: Back-End/src/main/java/com/flight_search/mapper/FlightMapper.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package com.flight_search.mapper;
+
+import com.flight_search.dto.FlightDTO;
+import com.flight_search.model.Flight;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface FlightMapper {
+    FlightMapper INSTANCE = Mappers.getMapper(FlightMapper.class);
```
Comment (mayralgr) [inline Back-End/src/main/java/com/flight_search/mapper/FlightMapper.java]: why name is INSTANCE ?

File: Back-End/src/main/java/com/flight_search/mapper/SegmentMapper.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
+package com.flight_search.mapper;
+
+import com.flight_search.dto.SegmentDTO;
+import com.flight_search.model.Flight.Segment;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface SegmentMapper {
+    SegmentMapper INSTANCE = Mappers.getMapper(SegmentMapper.class);
```
Comment (mayralgr) [inline Back-End/src/main/java/com/flight_search/mapper/SegmentMapper.java]: same here why INSTANCE?

also is INSTANCE used?

File: Back-End/src/main/resources/application.properties [hunk 5]
```diff
@@ -1 +1,7 @@
 spring.application.name=Back-End
+server.port=9090
+
+amadeus.api.key=Q7tnYFeoMtIJyxaGr3n0U75T6c7kvxTr
```
Comment (mayralgr) [inline Back-End/src/main/resources/application.properties]: this should be here, try to send this as an env variable to the container of find a safer way to do it. sensitive info should be in the repository

**Review by mayralgr — CHANGES_REQUESTED** [review]

File: Back-End/Dockerfile [hunk 6]
```diff
@@ -1,14 +1,27 @@
-# Usa una imagen base de OpenJDK 17
-FROM openjdk:17-jdk-alpine
+# Etapa de construcción
```
Comment (mayralgr) [inline Back-End/Dockerfile:1]: this is nos building correctly check https://github.com/ReBy298/FlightSearch/pull/3/files#diff-022779686499ed2a0c5b0a37c5d53a6f7b45627efaca9537b396459721c46f86

File: docker-compose.yml (hunk 1 above)
Comment (mayralgr) [reply docker-compose.yml:10]: this container name is not being applied, you need to add it with 

image: <name of the image>

File: Back-End/src/main/java/com/flight_search/mapper/FlightMapper.java (hunk 3 above)
Comment (ReBy298) [reply Back-End/src/main/java/com/flight_search/mapper/FlightMapper.java]: Stop using mappers and instead only use DTOs to parse the flight information.

File: Back-End/src/main/java/com/flight_search/mapper/SegmentMapper.java (hunk 4 above)
Comment (ReBy298) [reply Back-End/src/main/java/com/flight_search/mapper/SegmentMapper.java]: Stop using mappers and instead only use DTOs to parse the flight information.

File: Back-End/src/main/resources/application.properties (hunk 5 above)
Comment (ReBy298) [reply Back-End/src/main/resources/application.properties]: Fixed with environment variables

File: Back-End/Dockerfile (hunk 6 above)
Comment (ReBy298) [reply Back-End/Dockerfile:1]: Changed to FROM openjdk:17-jdk-alpine AS build

**Pull request merged**